setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="datamonster_api",
-    version="0.0.1",
+    version="0.0.2",
```

Comment:
This code looks fine, I'm just leaving this comment here to double check whether we handle versioning semver style or not.